### DISCUSSION TITLE
Getting error while we try to run cz command "cz bump", it was working fine till few days back.

### DISCUSSION THREAD

**Discussion by max-codex** [commitizen-tools/commitizen#756] [Q&A]
Hi All,
I am getting below error while running `cz bump` command

`usage: cz [-h] [--debug] [-n NAME] [--version]
          {ls,commit,c,example,info,schema} ...
cz: error: invalid choice: 'bump' (choose from 'ls', 'commit', 'c', 'example', 'info', 'schema')`

Do we have any such change which can prohibit us running `cz bump` command?
It was working fine earlier. If we have any change then what we need to do instead of this command to increment the version number. Please help me.

**Comment by woile** [comment]
`cz bump` has not been changed. What happens when you update to the latest version and run `cz bump --help`?

**Reply by max-codex** [reply]
This command is being used in my cicd pipeline 's yml file as 
``cz bump --dry-run | grep -i 'increment detected' | awk -F: '{print $2}'``
but as soon as it runs, it gives above error and fails the pipeline.

**Reply by woile** [reply]
Are you using the [github action](https://github.com/commitizen-tools/commitizen-action) or the [docker-image](https://github.com/commitizen-tools/commitizen-docker-image)? The docker image has had a breaking change, see [migration steps](https://github.com/commitizen-tools/commitizen-docker-image#migrating-from-v2-to-v3-image), that could be the issue (it's the only thing I can think of with the given info)

**Reply by max-codex** [reply]
cz is getting installed properly while image is being build but still it is giving same error.

**Reply by woile** [reply]
and which version is being installed? You may need to provide a bit more context to help. 

1. Which CI are you using?
2. Which OS are you using?
3. Are you using any of the provided docker images or installing from `pip` inside your container?
4. How are you installing it if using `pip`?

I just installed the latest version and `cz --help` shows `bump`

**Reply by max-codex** [reply]
1. I am using Gitlab with Kubernates executer
2. Using Kubernetes executor with custom docker image
3. installing using pip inside virtual env with the help of command `pip3 install commitizen`

**Reply by woile** [reply]
Mmm I'm really not sure, then. What you are using seems standard, and there shouldn't be any issue. Without a way to debug it properly there's not much more I can help with.

Some thoughts:
- try to run it locally, and see if the issue persists
- check the version with `cz version` (locally and in the docker image)
- try publish a new image to the docker registry

**Reply by max-codex** [reply]
For now, I got it fixed by reverting to older python image. cz seems to have issues with the newly created image only. I am debugging it further for the root cause. thanks everyone for help.
